lib/usd/utils/CMakeLists.txt

```diff
@@ -75,3 +83,74 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib OPTIONAL
     )
 endif()
+
+# -----------------------------------------------------------------------------
+# python
+# -----------------------------------------------------------------------------
+
+set(PY_INIT_FILES
+  ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py
+)
+
+# copy init files to build location for module tests
+foreach(INPUT_FILE ${PY_INIT_FILES})
+  string(REPLACE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME} OUTPUT_FILE ${INPUT_FILE})
+  execute_process(COMMAND ${CMAKE_COMMAND} -E copy ${INPUT_FILE} ${OUTPUT_FILE})
+endforeach()
+
+
+list(APPEND mayausdutils_python_source
+  module.cpp
+  wrapMayaTransformAPI.cpp
+)
+
+add_library(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  SHARED
+    ${mayausdutils_python_source}
+)
+
+target_compile_definitions(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  PRIVATE
+    MFB_PACKAGE_NAME=${TARGET_NAME}
+    MFB_ALT_PACKAGE_NAME=${TARGET_NAME}
+    MFB_PACKAGE_MODULE=${TARGET_NAME}
+)
+
+target_include_directories(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME} 
+    PUBLIC
+    ${UTILS_INCLUDE_LOCATION} 
+    ${MAYAUSDUTILS_INCLUDE_LOCATION} 
+    ${USD_INCLUDE_DIR}
+    )
+
+target_link_libraries(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  ${TARGET_NAME}
+  ${python_LIBRARIES}
+  usdGeom
+)
```

Comment:
No keyword? This is a recipe for disaster.  `build `and `usage `requirement should be clear.

Please explicitly specify PRIVATE, PUBLIC, INTERFACE here.